core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/SegmentIndexFileStore.java

```diff
@@ -162,6 +165,49 @@ public void readAllIIndexOfSegment(SegmentFileStore.SegmentFile segmentFile, Str
     }
   }
 
+  /**
+   * Get all index files list
+   *
+   * @param segmentFile
+   */
+  public List<CarbonFile> getAllIIndexOfSegment(SegmentFileStore.SegmentFile segmentFile,
```

Comment:
please remove this method, it has duplicate code, please reuse `org.apache.carbondata.core.metadata.SegmentFileStore#getIndexCarbonFiles`